homeassistant/components/downloader/__init__.py

```diff
@@ -43,6 +43,13 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     if DOMAIN not in config:
         return True
 
+    hass.async_create_task(_async_import_config(hass, config), eager_start=True)
```

Comment:
I don't think eager start makes sense since the component needs to be setup before the config entry is setup, and the config entry will be setup at the end of the config flow. We know that the task will need to yield to the loop to wait for the component setup to complete if we use an eager task. The whole point of this task is that we want it to run in a later iteration of the event loop to avoid a deadlock.

Reply:
Bdraco had it in his comment, do we want to remove it before the hotfix cut?